External/ietl/eigeninterface.h

```diff
@@ -40,8 +40,14 @@ namespace ietl {
             const Eigen::Matrix<TCoeffs, Eigen::Dynamic, 1>& x,
             Eigen::Matrix<TCoeffs, Eigen::Dynamic, 1>& y)
   { y = a*x; }
-
-
+  
+  template <class matrix_t, class TCoeffs>
+  void mult(const matrix_t& a,
```

Comment:
This modifies the source file of an external dependency, which can become a problem when updating the dependencies. (It can also be confusing if our version of IETL is subtly different from the original withour clear documentation.) Also note that in the current master the external dependencies are not stored in the repository anymore, see PR #70.

You could move the `ietl::mult` function to `AbstractMatrixWrapper.hpp`, where the matrix wrapper base class is defined.